### ISSUE TITLE
Build failure: libflux (from influxdb2)

### ISSUE THREAD

**Issue by poperigby** [NixOS/nixpkgs#319973]
### Steps To Reproduce

1. Set `service.scrutiny.enable = true`

### Build log

https://gist.github.com/poperigby/5b396cbcacfad0d1f4a3f5ef2f63e52d

### Additional context

There appears to be a cache miss, because `libflux` starts to be compiled, but it fails.

### Notify maintainers

@jnsgruk 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.33, NixOS, 24.11 (Vicuna), 24.11.20240613.e9ee548`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.22.1`
 - nixpkgs: `/nix/store/qqwr649pc0qprc9lw2fmdsi1km6p7q2h-source`
```
---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by eclairevoyant** [comment]
the maintainers to tag would be @abbradar for v2 and @offlinehacker and @zimbatm for v1 (all share the same build failure). v2 will be addressed via #319969

**Comment by poperigby** [comment]
Apologies. Thank you.
